lib/modules/sourceSnudown.js

```diff
@@ -95,7 +83,7 @@ addModule('sourceSnudown', function(module, moduleID) {
 					var thisResponse = JSON.parse(response.responseText),
 						sourceText = null,
 						userTextForm = $('<div class="usertext-edit viewSource"><div><textarea rows="1" cols="1" name="text" readonly></textarea></div><div class="bottom-area"><div class="usertext-buttons"><button type="button" class="cancel">hide</button></div></div></div>');
```

Comment:
You should probably just attach a `$` in front of the variable name to distinguish it as a jQuery variable.